### PR TITLE
Add flag for setting pytorch max_split_size_mb

### PR DESCRIPTION
**Pull request by priyakasimbeg**
The conformer pytorch workload still may run out of memory in current state. 
Add `reduce_pytorch_max_split_size` as a temporary workaround in case people into this issue. This will set 'max_split_size_mb:256'. Note that this will adversely impact the performance of the submission on this workload. See [tracking issue](https://github.com/mlcommons/algorithmic-efficiency/issues/497). 
Also minor formatting changes in other files.

### CODE REVIEW

File: submission_runner.py [hunk 1]
```diff
@@ -149,6 +149,9 @@
     None,
     'Value of rng seed. If None, a random seed will'
     'be generated from hardware.')
+flags.DEFINE_boolean('set_pytorch_max_split_size',
+                     None,
```
Comment (pomonam) [inline submission_runner.py:153]: Nit: We can set the default to `False`.

File: submission_runner.py [hunk 2]
```diff
@@ -602,6 +605,9 @@ def main(_):
   if FLAGS.workload == 'librispeech_conformer':
     os.environ['XLA_PYTHON_CLIENT_MEM_FRACTION'] = '0.85'
 
+  if FLAGS.set_pytorch_max_split_size is True:
```
Comment (pomonam) [inline submission_runner.py:608]: Nit: Also, we can set `if FLAGS.set_pytorch_max_split_size: ...`.

**Review by pomonam — APPROVED** [review]
LGTM!

File: submission_runner.py (hunk 1 above)
Comment (priyakasimbeg) [reply submission_runner.py]: Yup you're right that's better. Thanks!

File: submission_runner.py (hunk 2 above)
Comment (priyakasimbeg) [reply submission_runner.py]: Fixed, thanks!

**Pull request merged**